lib/OpenQA/Downloader.pm

```diff
@@ -22,6 +22,7 @@ use Mojo::File 'path';
 use Mojo::URL;
 use OpenQA::Utils 'human_readable_size';
 use Try::Tiny;
+use Time::HiRes 'sleep';
```

Comment:
What's this about? Added by accident? Or was this related to a bug you were investigating?

Reply:
Well, I guess this can help to fix sub-seconds sleep in some cases.

Reply:
I want to ensure that the sleep within the code further down supports sub-seconds.

Reply:
Ah, right. I didn't see any added sleeps with fractional numbers. Not a big fan of magic like that...